lib/manager/bitbucket-pipelines/extract.ts

```diff
@@ -0,0 +1,63 @@
+import { logger } from '../../logger';
+import { getDep } from '../dockerfile/extract';
+import type { PackageDependency, PackageFile } from '../types';
+
+export function extractPackageFile(content: string): PackageFile | null {
+  const deps: PackageDependency[] = [];
+
+  try {
+    const lines = content.split('\n');
+    for (let lineNumber = 0; lineNumber < lines.length; lineNumber += 1) {
+      const line = lines[lineNumber];
+
+      const pipeMatch = /^\s* - pipe:\s*'?"?([^\s'"]+)'?"?\s*$/.exec(line);
+      if (pipeMatch) {
+        const pipe = pipeMatch[1];
+        const pipeSplit = pipe.split(':');
+        const depName = pipeSplit[0].trim();
+        const currentValue = pipeSplit[1].trim();
+
+        const dep: PackageDependency = {
+          depName,
+          currentValue,
+          datasource: 'bitbucket-tags',
+        };
+
+        logger.debug(
```

Comment:
```suggestion
        logger.trace(
```
Too much noise otherwise